src/bin/bench.rs

```diff
@@ -64,7 +64,7 @@ Options:
     -p SPEC, --package SPEC ...  Package to run benchmarks for
     --all                        Benchmark all packages in the workspace
     --exclude SPEC ...           Exclude packages from the benchmark
-    -j N, --jobs N               Number of parallel jobs, defaults to # of CPUs
+    -j N, --jobs N               Number of parallel jobs, defaults to # of threads
```

Comment:
"hardware threads" maybe?
Just "threads" sounds even more ambiguous than "CPUs".

Reply:
Sounds good to me, I'll wait a bit before updating in case there are more suggestions :)